gokart/task.py

```diff
@@ -86,6 +90,9 @@ def __init__(self, *args, **kwargs):
         self._rerun_state = self.rerun
         self._lock_at_dump = True
 
+        if self.complete_check_at_run:
+            self.run = task_complete_check_wrapper(run_func=self.run, complete_check_func=self.complete)
```

Comment:
@mski-iksm  What will the log look like?

I'm wondering if we should add a log that shows what I skipped.

Reply:
@yokomotod 
I've added a log as following: 
https://github.com/m3dev/gokart/pull/333/files#diff-1c658ac433692b62825bf8604431590512f2ae636030e2041aa457858b486aaeR11